### PR TITLE
Enabling use of VS Code for debugging

### PR DESCRIPTION
**Pull request by mechaffin**
# Description and Motivation 

Enabling use of VS Code for debugging. Improve experience/documentation.

# Change type 

- [x] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

**Checklist:**

- [x] All tests passed
- [x] The code follows the code style and conventions of this project
- [x] The change requires a change to the documentation
- [x] I have updated the documentation accordingly

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "env": {
+                // Note: Any setting made in this file will override the operating system environment variable of that name.
+                // See https://docs.microsoft.com/aspnet/core/fundamentals/environments
+                "ASPNETCORE_ENVIRONMENT": "Development",
+                // Environment variables used in appsettings.ini.  
+                // Required Settings. 
+                "PCS_AZUREMAPS_KEY": "your Azure Maps key",
```
Comment (zhang-hua) [inline .vscode/launch.json]: use 'static' here?

**Comment by zhang-hua** [comment]
@mechaffin , one more thing need to do after we merge this PR. we might still need to add .vscode in .gitignore to avoid commit this file with credentials by mistake.

File: README.md [hunk 2]
```diff
@@ -133,31 +132,39 @@ required to package the service into a Docker image:
...
+import credentials and network details. Environment variables are not
+mandatory though, you can for example edit appsettings.ini and write
+credentials directly in the file. Just be careful not sharing the changes,
+e.g. sending a Pull Request or checking in the changes in git.
+
+The configuration file in the repository references some environment
+variables that need to be defined. Depending on the OS and the IDE used,
+there are several ways to manage environment variables.
+
+1. If you're using Visual Studio or Visual Studio for Mac, the environment
```
Comment (ppathan) [inline README.md]: nit: `Visual Studio for Windows or Mac`

File: README.md [hunk 3]
```diff
@@ -75,26 +75,25 @@ variables [here](#configuration-and-environment-variables).
 *  `PCS_AZUREMAPS_KEY` - the [Azure Maps](https://azure.microsoft.com/services/azure-maps/) 
   API Key. This can be set to "static" if you do not have one.
 
-## Running the service with Visual Studio
+## Running the service with Visual Studio or VS Code
+
```
Comment (molly-moen) [inline README.md]: should this be two separate sections? Since there are different pre-reqs for visual studio and vs code

Comment (mechaffin) [reply README.md]: Ug, maybe. I was following one of the other repos that grouped them together.  That's where I started with this whole task... across a gazillion repos.  I'd prefer to leave them together for now.  When the repos are merged, it'll be so much easier to change it then if we still think they shoudl be separate.

**Pull request merged**